test/e2e/tests/account-details.spec.js

```diff
@@ -248,7 +248,8 @@ describe('Show account details', function () {
           '[data-testid="account-list-item-menu-button"]',
         );
         await driver.clickElement('[data-testid="account-list-menu-pin"]');
-        assert.equal(await pinnedIcon.isDisplayed(), true);
+        const exists = await driver.isElementPresent(pinnedIcon);
+        assert.equal(exists, false, 'Unpinned Account');
```

Comment:
Have you tried `driver.waitForElementNotPresent()`?

Reply:
Thanks @HowardBraham, really nice suggestion!